content/en/docs/tasks/access-api-from-pod/access-api-from-pod.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Accessing the Kubernetes API from a Pod
+reviewers:
+- sftim
```

Comment:
I'd omit listing _any_ reviewers here. It's definitely not the case that I'm the only person who should review changes to this page.
```suggestion
```

Reply:
Done